doc/man7/OSSL_PROVIDER-serialization.pod

```diff
@@ -0,0 +1,99 @@
+=pod
+
+=head1 NAME
+
+OSSL_PROVIDER-serialization - OpenSSL serialization provider
```

Comment:
Base?...and so on in this file (and the file name is wrong too)

Reply:
Fixed the file name.